NitroxServer/Serialization/World/PersistedWorldData.cs

```diff
@@ -55,16 +37,16 @@ public bool IsValid()
                 return false;
             }
 
-            return (ParsedBatchCells != null) &&
-                   (ServerStartTime != null) &&
+            return (WorldData.ParsedBatchCells != null) && // Always returns false on empty saves
```

Comment:
Maybe add method `IsValid()` to WorldData class as well so this gets shorter and easier to read.

Code then becomes:
```
return WorldData.IsValid() && BaseData != null && ...
```